s3_sync/s3_sync.py

```diff
@@ -151,6 +151,26 @@ def sync_local_to_s3(local_video_records_done_folder, s3_bucket_name):
     logger.info("Syncing complete")
 
 
+def move_files_to_synced_folder(local_video_records_done_folder, local_video_records_synced_folder):
```

Comment:
This should probably operate on a list of files that were successfully synced, otherwise we run the risk of moving a video file out of the source directory when it hasn't yet been synced to S3. This could happen because of a sync failure (partial or total), or because the file was written to disk in the time between when the sync started and when it completes.